File: workspace_presentacion/Flotix2021/Flotix2021/Model/Alquiler.cs

namespace Flotix2021.Model
{
    /// <summary>
    /// Objeto Alquiler para mandar al servidor
    /// </summary>
    class Alquiler
    {
        public string idVehiculo { get; set; }
        public string idCliente { get; set; }
        public string fechaInicio { get; set; }
        public string fechaFin { get; set; }
        public int km { get; set; }
        public string tipoKm { get; set; }
        public string importe { get; set; }
        public string tipoImporte { get; set; }
	}
}

File: workspace_movil/app/src/main/java/com/flotix/model/Alquiler.kt
package com.flotix.model

import java.io.Serializable

class Alquiler(
    var idVehiculo: String = "",
    var idCliente: String = "",
    var fechaInicio: String = "",
    var fechaFin: String = "",
    var km: Int = 0,
    var tipoKm: String = "",
    val importe: Double = 0.0,
    var tipoImporte: String = ""
) : Serializable {
}